deep_speech_2/data_utils/data.py

```diff
@@ -58,6 +60,7 @@ def __init__(self,
                  window_ms=20.0,
                  max_freq=None,
                  specgram_type='linear',
+                 num_threads=12,
```

Comment:
The default value of num_thread  can be setted to [multiprocessing.cpu_count()](https://docs.python.org/2/library/multiprocessing.html#multiprocessing.cpu_count)?